test/unit/statement-builder.tests.js

```diff
@@ -21,21 +21,27 @@ var artistEntity = require(path.join(fixturesDir, 'artist-entity'));
 describe('StatementBuilder', function () {
   var schema = new Schema('artist', schemas.artist);
   var builder = new StatementBuilder(schema);
+  var fieldList = '"artist_id", "name", "create_date", "update_date", "members", "related_artists", "traits", "metadata"';
```

Comment:
we should be able to build this from the `Schema`, maybe `schema.fieldsString()`?